api/TextTrackList.json

```diff
@@ -101,7 +104,10 @@
       "change_event": {
```

Comment:
The event and the property were added at the same time in WebKit and Blink:
https://trac.webkit.org/changeset/159714/webkit
https://chromium.googlesource.com/chromium/src/+/1947d7da50d5bb254bc2baae2609cc9d06d6cfff